diff --git a/Dockerfile.debian b/Dockerfile.debian
index 7ee5dd61..9a184d5f 100644
--- a/Dockerfile.debian
+++ b/Dockerfile.debian
@@ -157,7 +157,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     libtool-bin                   \
     libwebp-dev                   \
     make                          \
-    libssl-dev                    \
     libpango1.0-dev               \
     libpulse-dev                  \
     uuid-dev                      \
